conn.go

```diff
@@ -739,6 +742,7 @@ func (c *Conn) handleBdat(arg string) {
 		}()
 	}
 
+	c.bdatReader.DisableCheck = true
```

Comment:
We need to restore the check after `BDAT` has been handled.

Maybe we can just save/restore `LineLimit` instead?

```go
prevLineLimit := c.bdatReader.LineLimit
c.bdatReader.LineLimit = 0

// Handle BDAT...

c.bdatReader.LineLimit = prevLineLimit
```

Then no need to have `DisableCheck`.

Reply:
When bdat read chunk we got curLineLength bigger than LineLimit for example LineLimit is 0 it not fix problem. May be change
```
if r.curLineLength > r.LineLimit && !r.DisableCheck {
```
to 

```
if r.curLineLength > r.LineLimit && r.LineLimit > 0 {
```

Reply:
Oh right. I guess you found another bug in our implementation, good catch!

Reply:
Okey i changed code like this. Add restore LineLimit after handle bdat